File: core/src/test/java/com/pluxity/TestApplication.java
package com.pluxity;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {"com.pluxity"})
public class TestApplication {
    public static void main(String[] args) {
        SpringApplication.run(TestApplication.class, args);
    }
}

File: common/src/test/java/com/pluxity/TestApplication.kt
package com.pluxity

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication(scanBasePackages = ["com.pluxity"])
object TestApplication {
    @JvmStatic
    fun main(args: Array<String>) {
        SpringApplication.run(TestApplication::class.java, *args)
    }
}
